examples/tests/valid/structs.test.w

```diff
@@ -70,4 +70,16 @@ test "struct definitions are phase independant" {
   };
 
   assert(s2.a == "foo");
-}
\ No newline at end of file
+}
+
+let x = (): str => {
+  struct PreflightStruct {
+    name: str;
+  }
+};
+
+let y = inflight (): str => {
+  struct InflightStruct {
+    name: str;
+  }
+};
```

Comment:
This file is in the `valid` test folder, so it should only contain passing tests. These failing cases should be moved to `examples/tests/invalid/structs.test.w`

Reply:
Moved the test cases to the proper ```invalid``` folder